### DISCUSSION TITLE
Missing device type

### DISCUSSION THREAD

**Discussion by vdt2210** [make-all/tuya-local#2192] [Q&A]
Hi there, after updating to 2024.7.1 I recognize the add-on has removed these device types. The type in the img is the best type for my smart plug, it's shows all the plug entities. What is the new device type to replace this? pls let me know, thanks

![image](https://github.com/user-attachments/assets/7f5c499d-ecb1-4e33-800a-e7b5488c0b96)

**Comment by make-all** [comment]
These types were not removed.

**Reply by vdt2210** [reply]
my mistake. After rechecking, I found these types again.
